### ISSUE TITLE
Suggestion: Add pistons to mech animation

### ISSUE THREAD

**Issue by Cpt-Fortius** [tsunamayo/Starship-EVO#6050]
Cheers!

The game keeps getting better and better!

Could you add pistons to mech animations, so they can be used the same way as rotators and hinges (label & set up movement / timing in mech computer)?

**Comment by najlitarvan** [comment]
while not directly mech, this animation system could potentially be used to do analog values if we could input multiple uttons and assign each a different thing to do

**Comment by tsunamayo** [comment]
@Cpt-Fortius @najlitarvan Yes this is something I had in mind, it is a bit tricky for mech as there are other things computed. But I wanted a "animation gate" a bit similar to make, to synchronise things.
